### DISCUSSION TITLE
DavX Contacts Not Showing in Fossily Contacts

### DISCUSSION THREAD

**Discussion by pixel24** [FossifyOrg/General-Discussion#681] [General]
Hello everyone,

I have connected my Pixel 8 Pro (GrapheneOS) to my Open-Xchange account via DavX. All contacts are available in the standard address book. After that, I installed Fossily Contacts and granted all permissions. However, no contacts are shown in this app, and the remote address book does not appear in the settings either.

The same procedure worked perfectly with my old Pixel 4 (GrapheneOS).

What am I doing wrong?

with best

**Comment by Aga-C** [comment]
It's already reported here: https://github.com/FossifyOrg/Contacts/issues/83.
